apps/web/components/ImageUpload.module.css

```
@@ -0,0 +1,109 @@
+  line-height: var(--type-label-lh);
+  color: var(--text-secondary);
+}
+
+.input {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)